src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    Optional<List<Order>> getByUser(User user);
```

Comment:
```suggestion
    List<Order> getByUser(User user);
```
We do not use Optional with collections. Optional is null-safe. The collection can be empty - no null problem.